src/lib_c/x86_64-windows-msvc/c/ntsysteminfo.cr

```diff
@@ -0,0 +1,54 @@
+require "c/winnt"
+require "c/win_def"
+require "c/int_safe"
+
+lib LibC
+  fun NTGetSystemInfo = GetSystemInfo(system_info : SystemInfo*)
```

Comment:
Is there a reason for `NT` prefix? I don't think that should be there.

Reply:
This function should be in `sysinfoapi.cr` (equivalent to `sysinfoapi.h`).

Reply:
It's probably better to use [`GetNativeSystemInfo`](https://docs.microsoft.com/de-de/windows/desktop/api/sysinfoapi/nf-sysinfoapi-getnativesysteminfo), though, which is generally equivalent but also works as expected WOW64 (32-bit layer on 64-but Windows).

Reply:
Switched to using [GetNativeSystemInfo](https://github.com/crystal-lang/crystal/blob/b9b590442aa9d0afcce2cb16af0e65e8a846ae76/src/lib_c/x86_64-windows-msvc/c/sysinfoapi.cr#L6).